ports/esp32/machine_sdcard.c

```diff
@@ -244,7 +253,8 @@ STATIC mp_obj_t machine_sdcard_make_new(const mp_obj_type_t *type, size_t n_args
 
         // Stronger external pull-ups are still needed but apparently
         // it is a good idea to set the internal pull-ups anyway.
-        // slot_config.flags = SDMMC_SLOT_FLAG_INTERNAL_PULLUP;
+        // test for 4line card
+        slot_config.flags = SDMMC_SLOT_FLAG_INTERNAL_PULLUP;
```

Comment:
This looks like an independent change not related to the freq addition.  Is that right?

Reply:
Restore it.